Examples/BookingSystem.AspNetCore/Stores/OrderStore.cs

```diff
@@ -287,7 +287,9 @@ public async override ValueTask CreateOrder(Order responseOrder, StoreBookingFlo
                 null,
                 null);
 
-            if (!result) throw new OpenBookingException(new OrderAlreadyExistsError());
+            if (!result) return CreateOrderResult.OrderAlreadyExists;
+
+            return CreateOrderResult.OrderSuccessfullyCreated;
```

Comment:
Noting that this abstraction - the idea of moving the throwing of the `OrderAlreadyExistsError` into the `StoreBookingEngine` - is excellent.

It is however, a breaking change for users of the library.

Will break this out into a separate issue to capture it for future.